src/SprykerEco/Zed/Payone/Communication/Plugin/Oms/Command/AbstractPayonePlugin.php

```diff
@@ -23,7 +24,7 @@ class AbstractPayonePlugin extends AbstractPlugin
      *
      * @return \Generated\Shared\Transfer\OrderTransfer
      */
-    protected function getOrderTransfer(SpySalesOrder $orderEntity)
+    protected function getOrderTransfer(SpySalesOrder $orderEntity): OrderTransfer
```

Comment:
It's a part of public API, please remove added type hints.
Please, chack other plugins as well

Reply:
Fixed.